mnd/mnd/models.py

```diff
@@ -205,3 +205,8 @@ class CarerRegistration(models.Model):
     registration_ts = models.DateTimeField(null=True, blank=True)
 
     objects = CarerRegistrationManager()
+
+
+class DuplicatePatient(models.Model):
+    patient = models.OneToOneField(Patient, related_name='duplicate_patient', on_delete=models.CASCADE)
+    is_duplicate = models.BooleanField(blank=False, null=False, default=False)
```

Comment:
Considered only having a `patient` field but I think this is the best way to do it given how demographics fields currently work